src/status_im/subs/wallet/networks.cljs

```diff
@@ -76,14 +76,13 @@
       network-values))))
 
 (re-frame/reg-sub
- :wallet/total-amount
+ :wallet/total-amount-in-to-chains
```

Comment:
We never used this subscriptions with `false` argument, so I removed it and renamed

Reply:
just a reminder that this sub will change after [this PR](https://github.com/status-im/status-mobile/pull/21116#issuecomment-2441267115)